cucumber-rails.gemspec

```diff
@@ -12,23 +12,23 @@ Gem::Specification.new do |s|
 
   s.license = 'MIT'
 
-  s.add_runtime_dependency('capybara', ['>= 1.1.2', '< 3'])
+  s.add_runtime_dependency('capybara', ['>= 1.1.2', '< 4'])
   s.add_runtime_dependency('cucumber', ['>= 3.0.2', '< 4'])
-  s.add_runtime_dependency('nokogiri', '~> 1.5')
-  s.add_runtime_dependency('railties', ['>= 4', '< 5.2'])
+  s.add_runtime_dependency('nokogiri', '~> 1.8')
+  s.add_runtime_dependency('railties', ['>= 4', '< 6'])
   s.add_runtime_dependency('mime-types', ['>= 1.17', '< 4'])
 
   # Main development dependencies
   s.add_development_dependency('ammeter', ['>= 1.0.0', '!= 1.1.3'])
   s.add_development_dependency('appraisal', '>= 0.5.1')
-  s.add_development_dependency('aruba', '~> 0.14.2')
+  s.add_development_dependency('aruba', '>= 0.14.2')
```

Comment:
The latest release of aruba is in the 0.14.x series, so this change has no effect currently.